connectd/connectd.c

```diff
@@ -358,7 +361,12 @@ struct io_plan *peer_connected(struct io_conn *conn,
 		 * in progress right now) */
 		if (connect->conn)
 			io_set_finish(connect->conn, NULL, NULL);
+		connect_reason = tal_steal(tmpctx, connect->reason);
```

Comment:
So an inbound connection when we were also wanting to connect to the peer will still be attributed with our connection request reason?